Certified Sail Extension (CSE) Examples/cse/mapField/v1/index.html

```diff
@@ -9,46 +9,49 @@
   <div id="map-container"></div>
   <script src='APPIAN_JS_SDK_URI'></script>
   <script>
-    var geocoder, currentLocation, map, pin, currentKey, scriptLoaded = false;
-    var newKey, newLocation;
+    var state = {};
+    var newKey, newLocation, currentKey;
+    var scriptLoaded = false, scriptId = 'scriptTag';
 
     // Handle a new user input location from SAIL.
     Appian.Component.onNewValue(function (newValues) {
       newKey = newValues.key;
       newLocation = newValues.location;
 
-      if (currentKey != newKey && newKey != "") {
-        if (currentKey == undefined) {
-          currentKey = newKey;
-          var s = document.createElement( 'script' );
-          s.setAttribute('src', "https://maps.googleapis.com/maps/api/js?key=" + newKey + "&callback=initMap");
-          document.body.appendChild( s );
-        } else {
-          Appian.Component.setValidations(["Changing the key is not supported"]);
-          document.getElementById('map-container').innerHTML = "";
-          return;
+      if (currentKey !== newKey && newKey !== "") {
+        scriptLoaded = false;
+        var oldScript = document.getElementById(scriptId);
+        if (oldScript) {
+          oldScript.parentElement.removeChild(oldScript);
+          document.getElementById("map-container").innerHTML = ""
```

Comment:
We should be consistent with quotes - probably worth switching these to use single quotes (`'`) instead